### ISSUE TITLE
Subcap Build Times

### ISSUE THREAD

**Issue by ScottyErata** [kabivel/Sins-of-Empires#81]
Many subcapital build times are not appropriate for their class. For example Caracals build is half the time of Cormorants.

**Comment by ScottyErata** [comment]
I'll look into it in the next week or so if nobody else picks up on it.

**Comment by Joeyxxx** [comment]
This was also raised last night.

**Comment by kabivel** [comment]
Do we have a list ?

**Comment by Joeyxxx** [comment]
I don't think this is a big deal for now. Personally, I don't even notice build times.
